inference-engine/tests_deprecated/unit/engines/gna/gna_matcher.cpp

```diff
@@ -274,6 +274,52 @@ void GNAPropagateMatcher :: match() {
                 *memoryAddress = &data.front();
                 return Gna2StatusSuccess;
             }));
+
+            EXPECT_CALL(mockApi, Gna2DeviceGetVersion(_,_)).WillOnce(Invoke([](
+                uint32_t deviceIndex,
+                enum Gna2DeviceVersion * deviceVersion) {
+                    *deviceVersion = Gna2DeviceVersionSoftwareEmulation;
+                    return Gna2StatusSuccess;
+                }));
+
+            EXPECT_CALL(mockApi, Gna2DeviceOpen(_)).WillOnce(Return(Gna2StatusSuccess));
+
+            EXPECT_CALL(mockApi, Gna2InstrumentationConfigCreate(_,_,_,_)).WillOnce(Return(Gna2StatusSuccess));
+
+            EXPECT_CALL(mockApi, Gna2ModelCreate(_,_,_)).WillRepeatedly(Invoke([](
+                uint32_t deviceIndex,
+                struct Gna2Model const * model,
+                uint32_t * modelId) {
+                    *modelId = 0;
+                    return Gna2StatusSuccess;
+                }));
+
+            EXPECT_CALL(mockApi, Gna2RequestConfigCreate(_,_)).WillRepeatedly(Invoke([](
+                uint32_t modelId,
+                uint32_t * requestConfigId) {
+                    *requestConfigId = 0;
+                    return Gna2StatusSuccess;
+                }));
+
+            EXPECT_CALL(mockApi, Gna2RequestConfigSetAccelerationMode(_,_)).WillRepeatedly(Return(Gna2StatusSuccess));
+
+            EXPECT_CALL(mockApi, Gna2RequestConfigEnableHardwareConsistency(_,_)).WillRepeatedly(Return(Gna2StatusSuccess));
+
+            EXPECT_CALL(mockApi, Gna2InstrumentationConfigAssignToRequestConfig(_,_)).WillRepeatedly(Return(Gna2StatusSuccess));
+
+            EXPECT_CALL(mockApi, Gna2RequestEnqueue(_,_)).WillRepeatedly(Invoke([](
```

Comment:
enque is a version of gna1 propgate forward i guess. If so move that into place where propagate forward was expected

Reply:
still waiting for that to be moved to correct case, for example that one do not expect propagate i guess. Having repeatedly() matche on enque is error prone - since some times it might be called say not 1 time, and this is not what test expected

``` c++
 case GnaPluginTestEnvironment::matchPrecision :
                        combined->add(new NNetPrecisionMatcher(_env.nnet_precision, INTEL_AFFINE));	                        combined->add(new NNetPrecisionMatcher(_env.nnet_precision, INTEL_AFFINE));
                        break;	                        
```